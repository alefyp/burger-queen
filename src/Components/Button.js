import React from 'react';
import styles from '../css/Button.module.css';

const Button = (props) => {
  return (
    <button type={props.type} className={styles.button} onClick={props.onClick}>{props.text}</button>
  );
}

export default Button;