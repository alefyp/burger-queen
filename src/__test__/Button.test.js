import React from 'react';
import renderer from 'react-test-renderer';
import Button from '../Components/Button';

it("Matches Button Component Snapshot", () => {
  const tree = renderer.create(<Button type="submit" text="Click me!" />).toJSON();
  expect(tree).toMatchSnapshot();
});




