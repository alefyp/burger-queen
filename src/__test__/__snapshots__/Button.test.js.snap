// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches Button Component Snapshot 1`] = `
<button
  className="button"
  type="submit"
>
  Click me!
</button>
`;
